platform/native/contiki-main.c

```diff
@@ -242,6 +242,8 @@ main(int argc, char **argv)
 
   serial_line_init();
 
+  process_start(&sensors_process, NULL);
```

Comment:
Without this, a call to sensors_changed() would not cause any event been generated and send to another process, I will add this to the commit message.